### PR TITLE
NT-1775:Refactor LoginToutActivity

### PR DESCRIPTION
**Pull request by hadia**
# 📲 What

Remove butter knife from LoginToutActivity and replaced with view binding

# 🤔 Why

deprecated butterknife

# 🛠 How

1- change LoginToutActivity from java to kotlin
2-remove butter-knife and replace it with viewbinding

# 👀 See

https://user-images.githubusercontent.com/1075310/108099462-4f6a5380-708d-11eb-993e-82d8f65b9e98.mp4

| Before 🐛 | After 🦋 |
| --- | --- |
|  |  |

# 📋 QA
open app try signup/login
# Story 📖

https://kickstarter.atlassian.net/browse/NT-1775

### CODE REVIEW

File: app/src/main/res/layout-land/login_tout_layout.xml [hunk 1]
```diff
@@ -1,74 +1,75 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView
```
Comment (Arkariang) [inline app/src/main/res/layout-land/login_tout_layout.xml:2]: What has changed on this file? I've reviewed and seem just identation changes, have I missed something @hadia

**Review by Arkariang — APPROVED** [review]
Lgtm!
I tested on my environment: log in with email and two factor auth, log in with email and without tfa,and sign up.

File: app/src/main/java/com/kickstarter/ui/activities/LoginToutActivity.kt [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            .compose(bindToLifecycle())
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe {
+                ViewUtils.showDialog(
+                    this,
+                    getString(R.string.general_error_oops),
+                    getString(R.string.login_tout_errors_facebook_authorization_exception_message),
+                    getString(R.string.login_tout_errors_facebook_authorization_exception_button)
+                )
+            }
```
Comment (leighdouglas) [inline app/src/main/java/com/kickstarter/ui/activities/LoginToutActivity.kt:62]: nit: add whitespace after line 62

**Review by leighdouglas — APPROVED** [review]
Looks great!

**Pull request merged**